<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainmine</class>
 <widget class="QMainWindow" name="Mainmine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>幼圆</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>简·教·学</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 246, 246);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>96</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="information_Layout">
     <item>
      <widget class="QLabel" name="ID">
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="comment">
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>备注</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name">
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_ID">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_comment">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_name">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="loginagain">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{ 
    border-radius:250px;
	background-color: rgb(123, 111, 255);
}
QPushButton:hover
{
	background-color: rgb(143, 141, 229);
	background-color: rgb(127, 147, 249);
	border-radius: 250px;
	
}</string>
    </property>
    <property name="text">
     <string>重新登录</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>200</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 147, 249);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="mode_homepage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>105</y>
       <width>200</width>
       <height>115</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{ 
    background-color:transparent;
}
QPushButton:hover
{
	background-color: rgb(127, 147, 249);
	border-radius: 0px;
	
}</string>
     </property>
     <property name="text">
      <string>首页</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mode_operation">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>200</width>
       <height>115</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{ 
    background-color:transparent;
}
QPushButton:hover
{
	background-color: rgb(127, 147, 249);
	border-radius: 0px;
	
}</string>
     </property>
     <property name="text">
      <string>操作</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mode_history">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>335</y>
       <width>200</width>
       <height>115</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{ 
    background-color:transparent;
}
QPushButton:hover
{
	background-color: rgb(127, 147, 249);
	border-radius: 0px;
	
}</string>
     </property>
     <property name="text">
      <string>历史
记录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mode_mine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>200</width>
       <height>115</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{ 
    background-color:transparent;
}
QPushButton:hover
{
	background-color: rgb(127, 147, 249);
	border-radius: 0px;
	
}</string>
     </property>
     <property name="text">
      <string>我的</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>200</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>简·教·学</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
